codalab/worker/worker_run_state.py

```diff
@@ -645,10 +699,17 @@ def _transition_from_FINALIZING(self, run_state):
         move on to the FINISHED state. Can also remove bundle_path now.
         """
         if run_state.is_restaged:
+            logger.info(f'{run_state.bundle.uuid} is restaged')
+            log_bundle_transition(
+                run_state.bundle.uuid, run_state.stage, RunStage.RESTAGED, 'it has finalized'
```

Comment:
This doesn't make sense to me because the log will state "...to RESTAGED due to it has finalized".